apps/rush-lib/src/logic/pnpm/IPnpmfileShimSettings.ts

```diff
@@ -4,6 +4,13 @@
 export interface IPnpmfileShimSettings {
   allPreferredVersions: { [dependencyName: string]: string };
   allowedAlternativeVersions: { [dependencyName: string]: ReadonlyArray<string> };
+  /**
+   * Path to `@rushstack/node-core-library`
+   */
+  coreLibraryPath: string;
```

Comment:
THis isn't used as far as I can tell, what's the purpose of including this?

Reply:
It's there for the sake of consumers writing their own PNPM files and wanting to be able to, for example, read rush.json and do special things when the packages in question are workspace projects.